tests/module-setup.sh

```diff
@@ -75,7 +75,7 @@ elif [[ $MACHINE_ID = gaea-c5 ]] ; then
         # /etc/profile here.
         source /etc/profile
     fi
-    source /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.sh
+    source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh
```

Comment:
@zach1221 @natalie-perlin my understanding is that we just need to do `module reset` here now.

Reply:
@ulmononian  good here